subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -80,6 +84,33 @@ static int publish_get_payload(struct mqtt_client *client,
 	return 0;
 }
 
+static int construct_job_id_get_topic(const u8_t *client_id,
```

Comment:
Don't use one line per arg, keep as many as possible on single line whilst not breaking 80 chars.

Reply:
only `const u8_t *suffix, u8_t *topic_buf)` could be made so that it won't break 80 chars. Looks weird but okei